pypeman/nodes.py

```diff
@@ -899,6 +899,67 @@ def generator(msg):
         return generator(msg)
 
 
+class MsgFuncNode(BaseNode):
+    """
+    Make a node from a simple function/coroutine that takes the message
+    and modifies it.
+
+    The given function can be async or not.
+    The name for the node will automatically be using the function's name.
```

Comment:
perhaps enhance this to.
"The name for the node will automatically be using the function's name." if no explicit value is specified.